web_development_101/project_html_css.md

```diff
@@ -93,8 +93,9 @@ There are a couple of ways to go about doing this, but the simplest is this:
 ### Student Solutions
 Submit a link below to the github repo with your files in it by using a pull request.  See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.
 
-* Add your solution below this list
```

Comment:
You need to change this line back to it's original format please. It will cause issues when it is rendered on the site with the modifications you've made. Thanks, Rhys